apps/condo/index.js

```diff
@@ -62,7 +56,13 @@ const keystone = new Keystone({
     },
 })
 
-if (!IS_BUILD_PHASE) {
+if (IS_BUILD_PHASE) {
```

Comment:
```suggestion
// Because Babel is used only for frontend to transpile and optimise code,
// backend files will bring unnecessary workload to building stage
if (IS_BUILD_PHASE) {
```